zh-Hans.lproj/Localizable.strings

```diff
@@ -406,12 +406,12 @@
 
 /* Assistive Mode */
 
-"PREFERENCES_GENERAL_ACCESSIBILITY_HEADER" = "Accessibility";
-"ASSISTIVE_DESCRIPTION" = "Assistive Mode is a simplified read-only user interface for Broadcasts designed for use with accessibility features like Assistive Access.\n\nEnabling this mode requires a relaunch of Broadcasts.";
-"ASSISTIVE_ENTER" = "Enter Assistive Mode";
-"ASSISTIVE_EXIT" = "Exit Assistive Mode";
-"ASSISTIVE_DIALOG_TITLE" = "Assistive Mode";
-"ASSISTIVE_DIALOG_NEXT_REGULAR" = "Broadcasts will now quit. On next launch, it will use its standard user interface.";
-"ASSISTIVE_DIALOG_NEXT_ASSISTIVE" = "Broadcasts will now quit. On next launch, it will use Assistive Mode.";
-"ASSISTIVE_DIALOG_QUIT" = "Quit";
-"ASSISTIVE_TOOLBAR_EXIT" = "Exit";
+"PREFERENCES_GENERAL_ACCESSIBILITY_HEADER" = "辅助功能";
+"ASSISTIVE_DESCRIPTION" = "在辅助模式下，Broastcasts拥有极简的用户界面，它为配合辅助访问等辅助功能而设计。\n\n开启此模式需要重启Broadcasts。";
```

Comment:
Typo here

Reply:
😨 Sorry about that. I've just corrected it.